publish.sh

```diff
@@ -11,5 +11,8 @@ set -eux
 cd "$(dirname "$0")/crates/macro"
 cargo publish
 
+# Let crates.io's index notice that we published the macro.
+sleep 10
```

Comment:
FWIW when I do this locally I just use `--no-verify` to work around this

Reply:
That works too, I suppose :-p